### ISSUE TITLE
Automate release tagging for `mozilla/rust-components-swift` github repo

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/application-services#4309]
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/SYNC-2678)
┆Epic: Unified a-s swift packages for iOS

**Comment by skhamis** [comment]
I completely agree with this! I think this is a great first step to simplify the release cutting process for this repo. I definitely agree with changing the tagging to stay with a-s as we're highly dependent on that anyways (and makes it simpler for consumers of this). 

Excited to see this come to fruition!
